sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session_receiver.py

```diff
@@ -151,7 +151,7 @@ def from_connection_string(
         :rtype: ~azure.servicebus.ServiceBusSessionReceiver
 
         :raises ~azure.servicebus.ServiceBusAuthenticationError: Indicates an issue in token/identity validity.
-        :raises ~azure.servicebus.ServiceBusAuthorizationError: Indicates an access/rights related failure.
+        :raises ~azure.servicebus.ServiceBusAuthorizationErrors an access/rights related failure.
```

Comment:
Looks like there's an 's' instead of a ':'?

Reply:
`_servicebus_session_receiver.py` doesn't exist any more after my unification PR merged in..